includes/Classifai/Services/Personalizer.php

```diff
@@ -0,0 +1,134 @@
+<?php
+/**
+ * Service definition for Language Processing
+ */
+
+namespace Classifai\Services;
+
+class Personalizer extends Service {
+
+	/**
+	 * Personalizer constructor.
+	 */
+	public function __construct() {
+		parent::__construct(
+			__( 'Recommended Content', 'classifai' ),
+			'personalizer',
+			[
+				'Classifai\Providers\Azure\Personalizer',
+			]
+		);
+	}
+
+	/**
+	 * Register the rest API endpoints
+	 */
+	public function init() {
+		parent::init();
+		add_action( 'rest_api_init', [ $this, 'register_endpoints' ] );
+		add_action( 'wp_ajax_render_recommended_content', [ $this, 'ajax_render_recommended_content' ] );
```

Comment:
We'll probably want to prefix these actions so we don't risk a conflict if some other plugin also uses the `render_recommended_content` action

Reply:
Done.